examples/unix/c11/z_get_attachment.c

```diff
@@ -68,12 +68,11 @@ void reply_handler(z_loaned_reply_t *reply, void *ctx) {
 
         // Check attachment
         const z_loaned_bytes_t *attachment = z_sample_attachment(sample);
-        if (attachment == NULL) {
-            return;
-        }
         ze_deserializer_t deserializer = ze_deserializer_from_bytes(attachment);
         size_t attachment_len;
-        ze_deserializer_deserialize_sequence_length(&deserializer, &attachment_len);
+        if (ze_deserializer_deserialize_sequence_length(&deserializer, &attachment_len) != 0) {
```

Comment:
```suggestion
        if (ze_deserializer_deserialize_sequence_length(&deserializer, &attachment_len) < 0) {
```
According to our API, an error is a value less than zero and in others examples we check "< 0"